packages/replayio/src/utils/recordings/upload/uploadRecording.ts

```diff
@@ -292,7 +291,6 @@ async function uploadRecordingReadStream(
   try {
     const response = await Promise.race([
       fetch(url, {
-        agent: getKeepAliveAgent,
```

Comment:
from what I understand, keep alive is already the default in `undici`:
https://github.com/nodejs/undici/blob/985b3816708512bafefa1544def183cc6d1536be/lib/client.js#L273-L274